pkgs/development/r-modules/default.nix

```
@@ -309,6 +309,7 @@ let
   };
 
   packagesWithNativeBuildInputs = {
+    asciicast = [ pkgs.lzma pkgs.bzip2 pkgs.zlib pkgs.icu ];
```

```suggestion
    asciicast = with pkgs; [ lzma.dev bzip2.dev zlib.dev icu.dev ];
```
I think this can work.

---

Just we use `with`? I was under the impression that this should be avoided in favour of explicitly citing the namespace.

---

AFAIK, one level of `with` is tolerated, especially if it improves readability. But nesting them is to be avoided. I don't insist on it, we can go without. How about using only the dev output of the derivations? I think that reduces the size of deps

---

yes, that's a good point, just did it :+1: